cmake/CMakeFiles/3.5.1/CMakeSystem.cmake

```diff
@@ -0,0 +1,15 @@
+set(CMAKE_HOST_SYSTEM "Linux-4.4.0-62-generic")
```

Comment:
whats this here for? Basicallly this whole directory CMakeFiles should not be here